piper/config.yaml

```diff
@@ -19,14 +19,18 @@ options:
   noise_scale: 0.667
   noise_w: 0.333
   max_piper_procs: 1
+  debug_logging: false
+  update_voices: false
```

Comment:
```suggestion
  update_voices: true
```

Reply:
OK, let me make the loading a bit more robust and I'll enable it by default.